bundles/k3d-test/uds-bundle.yaml

```diff
@@ -0,0 +1,30 @@
+kind: UDSBundle
+metadata:
+  name: k3d-core-test
+  description: A UDS bundle for deploying uds-k3d-dev for CI/test
+  version: test
+
+packages:
+  - name: uds-k3d-dev
+    repository: ghcr.io/defenseunicorns/packages/uds-k3d
+    # renovate: datasource=github-tags depName=defenseunicorns/uds-k3d versioning=semver
+    ref: 0.3.1
+    overrides:
+      uds-dev-stack:
+        minio:
+          values:
+            - path: "buckets"
+              value:
+                - name: "uds-loki-chunks"
+                - name: "uds-loki-rules"
+                - name: "uds-loki-admin"
+            - path: "users"
+              value:
+                - accessKey: logging
+                  secretKey: "logging-secret"
+                  policy: readwrite
```

Comment:
I'm wondering if it makes sense to just create a default user in uds-k3d. We already have a default bucket `test` and I think Loki can use the same bucket for all 3 storage pieces. If we made a default user/pass we could skip the need for this extra test bundle I think?

Reply:
you can just use the built in admin user generated by the chart. If you wanted to test using a role to connect using the overrides here would be a good option. I think for this use case if the test is going to be with  a single bucket writing to folders you can probably just use the generated admin creds for minio and the test bucket.

Totally a viable option to add some defaults to the uds-k3d stack though and it should be minimal overhead if they arent being used.

Reply:
Added an explicit user and bucket this [uds-k3d PR](https://github.com/defenseunicorns/uds-k3d/pull/41).  Not sure how I feel about adding hard-coded minio credentials over there but it would certainly make testing a lot easier by alleviating the need to configure the credentials/buckets with bundle overrides.  It seems like the least of all of the evils at this point.

@zachariahmiller I considered using the admin credentials but I was hoping to avoid creating tasks to 'setup' minio, or lookup credentials, plus I would not have anyway to set those credentials in the uds-core package unless I added ZARF_VARS.  Maybe there is an easy way to do that that I am not aware of but it seems like my choices are:

* pre-baked credentials for a minio user that are known ahead of time
* bundle overrides (I believe our ideal outcome is to avoid wrapping the single package tasks in a bundle)

Reply:
Quite the conundrum

Reply:
> pre-baked credentials for a minio user that are known ahead of time

I think this is the right answer since it simplifies everything to a point where we can _hopefully_ do package tests without a bundle. It should also help with other charts that need s3 similarly, to some extent, without having to use zarf actions/tasks in the middle of a bunch/deploy to grab existing users somehow.

Reply:
It's funny I was originally going to add that to the minio chart values and didn't because it was a package specific implementation of what was supposed to be a generic tool/stack. I think this is fine at the moment, hopefully we can get to a better way in the future.

Reply:
I think as long as we keep the user/bucket generic (currently `uds` in @rjferguson21 's PR there) it should be okay and generic enough for usage. Doesn't solve GitLab and other places where multiple buckets are needed, or handle fine-grained auth, but I think that's okay.